src/components/Faq/FaqAccordion.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { MinusIcon, AddIcon } from '@chakra-ui/icons'
+import {
+  Accordion,
+  AccordionItem,
+  Text,
+  AccordionButton,
+  Box,
+  AccordionPanel,
+  Flex,
+  Icon,
+} from '@chakra-ui/react'
+import { IconType } from 'react-icons'
+
+const SingleAccordion = ({
+  header,
+  body,
+}: {
+  header: string
+  body: string
+}) => {
+  return (
+    <Flex direction="column">
+      <Accordion defaultIndex={[1]} allowMultiple my={{ base: 1 }}>
+        <AccordionItem
+          borderRadius="5"
+          py={{ base: 5 }}
+          border="none"
+          _light={{ boxShadow: '0px 0px 20px 11px #80808012', bg: '#ffffff' }}
+          _dark={{ bg: '#2D3748' }}
+        >
+          {({ isExpanded }) => {
+            return (
+              <>
+                <Text
+                  fontWeight="bold"
+                  fontSize={{ base: 'sm', md: 'md', lg: 'xl' }}
+                >
+                  <AccordionButton
+                    _hover={{ bg: 'none' }}
+                    _focus={{ outline: 'none' }}
+                  >
+                    <Box flex="1" textAlign="left">
+                      <Text
+                        fontWeight="bold"
+                        fontSize={{ base: 'sm', md: 'md', lg: 'md' }}
+                      >
+                        {header}
+                      </Text>
+                    </Box>
+                    {/* <AccordionIcon /> */}
+                    {isExpanded ? (
+                      <MinusIcon fontSize="12px" />
+                    ) : (
+                      <AddIcon fontSize="12px" />
+                    )}
+                  </AccordionButton>
+                </Text>
+                <AccordionPanel pb={4}>
+                  <Text
+                    fontSize={{ base: 'sm', md: 'sm', lg: 'sm' }}
+                    lineHeight={{ base: '2' }}
```

Comment:
We don't need the responsive syntax, when the only breakpoint is base.
```suggestion
                    lineHeight={2}
```